cmds/core/hostname/hostname.go

```diff
@@ -15,13 +15,26 @@ import (
 	"fmt"
 	"log"
 	"os"
+
+	"golang.org/x/sys/unix"
 )
 
 func main() {
-	hostname, err := os.Hostname()
-	if err != nil {
-		log.Fatalf("could not obtain hostname: %v", err)
-	}
+	if len(os.Args) == 2 {
+		new_hostname := []byte(os.Args[1])
```

Comment:
nit Go uses camelcase

Reply:
Also no need to cast, it's already that

newHostname := os.Args[1]

Reply:
I've addressed the camelcase, thanks for pointing that out. The cast needs to stay, `unix.Sethostname()` wants `[]byte`, but I've moved it to the call site to make it clearer.